pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc

```diff
@@ -1,18 +1,24 @@
-= Policy Development Kit Overview
+= Flex Gateway Policy Development Kit (PDk) Overview
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
 Policies enforce regulations to help manage security, control traffic, and improve API adoption. Flex Gateway provides many xref:gateway::policies-included-directory.adoc[included policies], but they might not cover all your specific business needs.
 
-The Policy Development Kit (PDK) is a software development kit that enables Flex Gateway users to develop custom policies with ease.
+Flex Gateway Policy Development Kit (PDK) is a software development kit that enables Flex Gateway users to develop custom policies with ease.
 
-PDK provides step-by-step instructions to guide you through the policy development lifecycle, from implementing your Rust source code to releasing your policy on Exchange. Like included policies, you can apply custom policies to Flex Gateway by using API Manager or YAML configuration files.
+PDK provides instructions to guide you through the policy development lifecycle, from implementing your Rust source code to releasing your policy on Exchange. Like included policies, you can apply custom policies to Flex Gateway running in Connected Mode by using API Manager or to Flex Gateway running in Local Mode by using YAML configuration files.
 
-PDK also provides template policies to help you get started with policy development.
+PDK also provides xref:pdk::policies-pdk-develop-custom-policies.adoc[example policies] to help you get started with policy development.
 
-== Before You Begin
+Watch the PDK Spotlight video to see a quick overview of PDKß.
```

Comment:
```suggestion
Watch the PDK Spotlight video to see a quick overview of PDK.
```

Reply:
Also, is "Spotlight" supposed to be capitalized?

Reply:
oof, forgot to push